spec/commands_spec.rb

```diff
@@ -1,121 +1,128 @@
-require './spec/spec_helper'
+require 'spec_helper'
 
-describe "pry-rescue commands" do
-  describe "try-again" do
-    it "should throw try_again" do
-      PryRescue.should_receive(:in_exception_context?).and_return{ true }
+RSpec.describe "#Pry-rescue commands" do
+  describe "#try-again" do
+    it "expect to throw try_again" do
+      expect(PryRescue).to receive(:in_exception_context?).and_return(true)
 
-      lambda{
-        Pry.new.process_command "try-again", '', TOPLEVEL_BINDING
-      }.should throw_symbol :try_again
+      expect( lambda{ Pry.new.process_command "try-again TOPLEVEL_BINDING" } ).to throw_symbol :try_again
     end
 
-    it "should raise a CommandError if not in Pry::rescue" do
-      PryRescue.should_receive(:in_exception_context?).and_return{ false }
+    it "expect to raise a CommandError if not in Pry::rescue" do
+      # expect(PryRescue).to receive(:in_exception_context?).and_return(false)
+      allow(PryRescue).to receive(:in_exception_context?).and_return(false)
 
-      lambda{
-        Pry.new.process_command "try-again", '', TOPLEVEL_BINDING
-      }.should raise_error Pry::CommandError
+       expect{ Pry.new.process_command "try-again TOPLEVEL_BINDING" }.to raise_error Pry::CommandError
     end
   end
 
-  describe "cd-cause" do
-    it "should enter the context of an explicit exception" do
+  describe "#cd-cause" do
+    it "expect to not enter the context of an explicit exception" do
       begin
         b1 = binding
         raise "original"
-      rescue => e1
+      rescue => ErrorOne
```

Comment:
@magicalbanana This creates a new global constant, which you almost certainly don't want to do (it will leak between tests). If you need to extend the scope of the variable use a local that's set to nil at the start of the test. Same applies everywhere you are rescuing into a constant.